server/src/static/js/content-helper.js

```diff
@@ -1,19 +1,20 @@
 /*jslint browser: true */
+/* global POST_MESSAGE_ORIGIN */
```

Comment:
Here `POST_MESSAGE_ORIGIN` is actually something like `SITE_ORIGIN`, right?  That is, it should be `https://pageshot.dev.mozaws.net`, where above it would be `https://pageshotcontent.dev.mozaws.net`